layer1/executor/tasks/base_task.go

```diff
@@ -158,7 +127,12 @@ func (bt *BaseTask) GetAllowMultiExecution() bool {
 func (bt *BaseTask) GetSubscribeOptions() *transaction.SubscribeOptions {
```

Comment:
I understand what you mean, but we need the ability to send nil as `SubscribeOptions` because we can't rely on the internal values that will have default values.